task_02/novikov_dmitry/task_02.cpp

```diff
@@ -0,0 +1,242 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <map>
+#include <memory>
+#include <string>
+#include <fstream>
+#include <sstream>
+#include <set>
+
+class Vertex {
+private:
+    int vertex_id;
+public:
+    //other information
+    Vertex() : vertex_id(-1) {}
+    Vertex(int id) : vertex_id(id) {}
+    Vertex(const Vertex &item) : vertex_id(item.vertex_id) {}
+    //Vertex(int id, ...) {}
+    bool operator >(const Vertex &v) const {
+        return this->vertex_id > v.vertex_id;
+    }
+    bool operator <(const Vertex &v) const {
+        return this->vertex_id < v.vertex_id;
+    }
+    bool operator ==(const Vertex &v) const {
+        return this->vertex_id == v.vertex_id;
+    }
+    bool operator !=(const Vertex &v) const {
+        return this->vertex_id != v.vertex_id;
+    }
+
+    std::string vertex_to_str() const{
+        std::stringstream ss_out;
+        ss_out << this->vertex_id;
+        return ss_out.str();
+    }
+    void get_Info() const {
+        std::string str = std::string(4, ' ');

```

Comment:
Я так понимаю это круто выглядящий способ создания строки с 4мя пробелами?)
Чем тебе не нравится такая запись?
```suggestion
        std::string str = "    ";
```